### PR TITLE
Migrate soft-deprecated leex templates to HEEx

### PR DESCRIPTION
**Pull request by grzuy**

### CODE REVIEW

File: test/elixir_console_web/live/console_live_test.exs [hunk 1]
```diff
@@ -20,7 +20,7 @@ defmodule ElixirConsoleWeb.ConsoleLiveTest do
     end
 
     test "command result is displayed", %{html: html} do
-      assert html =~ ~r/<div id="output.".*3.*<\/div>/s
+      assert html =~ ~r/<div.*id="output.".*3.*<\/div>/s
```
Comment (eldano) [inline test/elixir_console_web/live/console_live_test.exs:23]: Just a detail but by making this change now we're not necessarily matching the id="output" inside the div tag anymore.
I tested it changing in the history_component the output div by a different tag (section) and it still passes.
To scope it to the same div tag we need to change `.*` by `[^>]*`.

**Review by eldano — APPROVED** [review]
Looking good! just some minor comments

File: lib/elixir_console_web/live/console_live/history_component.html.heex [hunk 2]
```diff
@@ -1,11 +1,10 @@
-<div class="flex-1"></div>
 <div class="p-2" id="commandOutput" phx-update="append">
   <div id="version-info" class="text-gray-500 font-medium">
     <p>Elixir <%= System.version() %>/OTP <%= System.otp_release() %></p>
   </div>
   <%= for output <- @output do %>
-    <div id="command<%= output.id %>" class="text-gray-300 font-medium"><%= print_prompt() %><%= format_command(output.command) %></div>
-    <div id="output<%= output.id %>" class="text-teal-300">
+  <div id={"command#{output.id}"} class="text-gray-300 font-medium"><%= print_prompt() %><%= format_command(output.command) %></div>
```
[the remark this answers is not in the document]
Comment (grzuy) [reply lib/elixir_console_web/live/console_live/history_component.html.heex:6]: I like that - thanks!

File: test/elixir_console_web/live/console_live_test.exs (hunk 1 above)
Comment (grzuy) [reply test/elixir_console_web/live/console_live_test.exs:23]: Good call!

**Pull request merged**